cypress/integration/Toggle.spec.js

```diff
@@ -0,0 +1,91 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
+describe('The Toggle component', () => {
+  beforeEach(() => {
+    cy.visit(
+      '/iframe.html?selectedKind=Action%7CToggle&selectedStory=basic%20toggle&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel'
+    );
+
+    // Add data-id attribute
+    cy.get('label > input').invoke('attr', 'data-id', 'toggle-input');
```

Comment:
This is interesting! Never even thought to do this.

I think it would be OK to add the data attribute in the Storybook file, just to keep things more consistent with other elements. In the app, for example, we do actually render out these attributes so they can interact with Pendo. Obviously we don't need analytics for our Storybook site, but this would just keep things more in line with expectations elsewhere.